test/integration/getserversideprops/pages/to-json.js

```diff
@@ -0,0 +1,21 @@
+class Post {
+  constructor(title) {
+    this.title = title
+  }
+
+  toJSON() {
+    return { title: this.title }
+  }
+}
```

Comment:
This class would serialize to the same string with `JSON.stringify(new Post('hello'))` whether it has this `toJSON` method or not.
To correctly test whether SSR called `.toJSON`, wouldn't you want it to be different? i.e. you'd have something like:
```js
  toJSON() {
    return { title: '.toJSON() was called' }
  }
```
and in the test check for:
```js
expect(html).toContain('.toJSON() was called')
```

Reply:
Good catch @Janpot!  Resolving now...

Reply:
Resolved via 65e5d0e944d14325827de05e70f5cf1ca6541409